### DISCUSSION TITLE
'And' - 'But' is declared but its value is never read.ts

### DISCUSSION THREAD

**Discussion by thanujamsc** [badeball/cypress-cucumber-preprocessor#1190] [General]
I'm new to Cypress and and able to install badeball for BDD in font end.

'And' and 'But' is declared but its value is never read.ts
Module '"@badeball/cypress-cucumber-preprocessor"' has no exported member 'And'.

Why we do not have And and But?

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/issues/821
